lua/modules/configs/tool/dap/dap-dlv.lua

```diff
@@ -38,6 +38,21 @@ end
 -- https://github.com/go-delve/delve/blob/master/Documentation/usage/dlv_dap.md
 dap.configurations.go = {
 	{ type = "go", name = "Debug", request = "launch", program = "${file}" },
+	{
+		type = "go",
+		name = "Debug with args",
+		request = "launch",
+		program = "${file}",
+		args = function()
+			local args = {}
+			local args_string = vim.fn.input("Args: ")
+			for word in args_string:gmatch("%S+") do
+				table.insert(args, word)
+			end
+			vim.notify(args)
+			return args
+		end,
```

Comment:
```suggestion
		args = function()
			local argument_string = vim.fn.input("Program arg(s): ")
			return vim.fn.split(argument_string, " ", true)
		end,
```
Vim has such function builtin 😄

Reply:
Great! PR updated.